contents/docs/self-host/deploy/hosting-costs.md

```diff
@@ -10,25 +10,49 @@ Charges on various platforms can be confusing to understand as loadbalancers (wh
 
 See [Hobby Deployment](hobby) for details on volumes.
 
-## DigitalOcean ($56/mo)
+## DigitalOcean
 
 See [DigitalOcean Kubernetes pricing](https://www.digitalocean.com/pricing#kubernetes).
 
 At the time of writing the suggested default setup is as follows:
-1. 40$ for compute (2x smallest prod nodes)
-1. 6.4$ for storage (64Gi block storage)
-1. 10$ for load balancer (1x small LB)
+1. $40 for compute (2x smallest prod nodes)
+1. $6.4 for storage (64Gi block storage)
+1. $10 for load balancer (1x small LB)
 
-Making the total 56.4$ / month.
+Making the total \~$60 / month
 
-## AWS ($100/mo)
+## AWS
 
 At the time of writing the suggested default setup is as follows:
-1. 70$ for compute (2x m5.large). Note that reserved instances are about 40% cheaper if you can commit to a year.
-1. 6.4$ for storage (64Gi block storage). Note that potentially you'll get charged for snapshots extra.
-1. 20$ for load balancer (18.25 + 0.008 / GB of data was processed)
+1. $70 for compute (2x m5.large). Note that reserved instances are about 40% cheaper if you can commit to a year.
```

Comment:
I think $70 is a single `m5.large` with on-demand pricing. For similar spec as DO we could suggest 2x `t3.medium` (2 vCPU with 4GB memory) = $59.90